src/device/device_ledger.cpp

```diff
@@ -1353,7 +1353,7 @@ namespace hw {
         this->exchange();
 
         //pseudoOuts
-        if ((type == rct::RCTTypeSimple) || (type == rct::RCTTypeSimpleBulletproof)) {
+        if (type == rct::RCTTypeSimple) {
```

Comment:
Are you sure you don't need a || type == rct::RCTTypeBulletproof ?

Reply:
If I have correctly understood, there is no PseudoOuts with bulletproof, no?

Reply:
There are pseudoOuts. They're moved to the prunable part though.

Reply:
I mean as there are moved in prunable, there are not anymore in  in rctSigBase.pseudoOuts.

Here, I re-serialize what have been done in serialize_rctsig_base and so perform the same conditional test as in https://github.com/monero-project/monero/blob/master/src/ringct/rctTypes.h#L257

Reply:
OK, I went to see the context and it looks OK, sorry )

Reply:
No worries, I prefer you ask rather than missing something. :)